dir.traversal.targets

```diff
@@ -195,9 +223,14 @@
 
   <Target Name="Rebuild" DependsOnTargets="Clean;Build" />
 
-  <Target Name="RestorePackages" DependsOnTargets="$(TraversalRestorePackagesDependsOn)" />
+  <Target Name="RestorePackages" Condition="'$(_BuildAgainstPackages)' != 'true'" DependsOnTargets="$(TraversalRestorePackagesDependsOn)" />
```

Comment:
The condition should be on the target it should be on the TraversalRestorePackagesDependsOn property instead.